src/test/java/marquez/api/JobResourceTest.java

```diff
@@ -56,7 +57,7 @@
   private static final NamespaceService MOCK_NAMESPACE_SERVICE = mock(NamespaceService.class);
   private static final JobResource JOB_RESOURCE =
       new JobResource(MOCK_NAMESPACE_SERVICE, MOCK_JOB_SERVICE);
-  private static final String NAMESPACE_NAME = "someNamespace";
+  private static final NamespaceName NAMESPACE_NAME = NamespaceName.fromString("test");
```

Comment:
we'll want a generator for `NamespaceName`s ... might be a good opportunity to start a `marquez.common.models.Generator`

Reply:
You're right. Let's open an epic around testing changes (i.e. clean up tasks for test suites, providing solid test examples for new comers to our codebase, etc). So, I'll hold off for now, but something we will be revisiting soon.